common-arcana.lic

```diff
@@ -147,7 +138,7 @@ module DRCA
     command = spell['prep'] if spell['prep']
     command = spell['prep_type'] if spell['prep_type']
 
-    return unless prepare?(spell['abbrev'], spell['mana'], spell['symbiosis'], command, tattoo_tm = false, spell['runestone_name'], spell['runestone_tm'])
```

Comment:
I also just noticed that you have `tattoo_tm = false`. It's probably safer to just use `false`.

Reply:
Also made it `spell['tattoo_tm'].